install.sh

```diff
@@ -11,7 +11,7 @@ esac
 
 if [ $# -eq 0 ]; then
 	deno_asset_path=$(curl -sSf https://github.com/denoland/deno/releases |
-		grep -o "/denoland/deno/releases/download/.*/deno_${os}_x64\.gz" |
+		grep -o "/denoland/deno/releases/download/.*/deno_${os}_x64\\.gz" |
```

Comment:
Intentional?

Reply:
Yes, a newer version of `shellcheck` reported this, and it's correct.